source/index.html.md

```diff
@@ -380,21 +395,58 @@ Example response (snippet):
     "status": {
         "code": 20000
     },
-    "processed_at": "2018-07-09T12:58:56+00:00",
-    "recurring": true,
-    "_embedded": { "card": { "id": "58dabba0-e9ea-4133-8c38-bfa1028c1ed2" } }
+    "processed_at": "2019-08-14T12:58:56+00:00",
+    "series": {
+        "type": "recurring",
+        "initiator": "cardholder"
+    },
+    "_embedded": {
+        "self": { "href": "/authorizations/04dce6bf-ca17-42c5-a61e-5fdb4b87310a" }
+    }
 }
 ````
 
 This should be followed by a capture except when the amount is `0`.
 
-Subsequent authorizations are made similarly, but neither CSC nor PARes (see
-[3-D Secure](#3-d-secure)) would be included.
+Subsequent authorizations are made by using `series[previous]` which point to
+the previous authorization in the series; for `installment`, `recurring`, and
+`unscheduled` where subsequent transactions are merchant-initiated, neither CSC
```

Comment:
I would like a comma between "`unscheduled`" and "where" due to the comma after "merchant-initiated" :slightly_smiling_face: :upside_down_face: